com.unity.render-pipelines.universal/Editor/Lighting/UniversalRenderPipelineLightUI.Drawers.cs

```diff
@@ -53,24 +64,198 @@ enum Expandable
         );
 
 
-        static void DrawGeneralContent(object serialized, Editor owner)
+        static void DrawGeneralContent(UniversalRenderPipelineSerializedLight serializedLight, Editor owner)
         {
+            serializedLight.settings.DrawLightType();
+
+            Light light = serializedLight.settings.light;
+            if (LightType.Directional != light.type && light == RenderSettings.sun)
+            {
+                EditorGUILayout.HelpBox(Styles.SunSourceWarning.text, MessageType.Warning);
+            }
+
+            if (!serializedLight.settings.lightType.hasMultipleDifferentValues)
+            {
+                using (new EditorGUI.DisabledScope(serializedLight.settings.isAreaLightType))
+                    serializedLight.settings.DrawLightmapping();
+
+                if (serializedLight.settings.isAreaLightType && serializedLight.settings.lightmapping.intValue != (int)LightmapBakeType.Baked)
```

Comment:
Area lights are baked only, that is why is forced. This was code from the old inspector. 

As this code is only executed when the Render Pipeline Asset is from URP I do not think it will collide on cross-pipeline project.